packages/dappmanager/test/integration/fetchSystemData.test.int.ts

```diff
@@ -45,13 +45,22 @@ describe("Get system data", () => {
     }
   }).timeout(10 * 1000);
 
-  it("Should get host machine stats", async () => {
-    const result = await calls.getStats();
-    // Can't check actual returns since they vary
-    expect(result.cpu).to.include("%");
-    expect(result.memory).to.include("%");
-    expect(result.disk).to.include("%");
-  }).timeout(10 * 1000);
+  it("Should return parsed disk stats from host machine", done => {
+    calls
+      .statsDiskGet()
```

Comment:
Use await syntax like the previous test.
Doing
```
.then(done, done);
```
means that either if the promise rejects or resolves the test passes